### DISCUSSION TITLE
Bear Watcher: .env not in cloned repo

### DISCUSSION THREAD

**Discussion by caseywalker** [nss-evening-cohort-14/evening-client-side#109] [Help Me!] [answered]
Casey Walker 

- Have you searched the other help tickets? Yes. 

- If there are similar issues, make sure to give us their number/link to them and tell how yours is different. N/A

- Describe your issue. **The Bear Walker Project does not include the .env file.**

- List any error messages that appear. No issues, wondering if we need the .env file. 

- What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
1. https://stackoverflow.com/questions/32773000/git-doesnt-push-env-file-on-the-server <--- Apparently can be ignored, or removed from git ignore. 
2. https://forum.freecodecamp.org/t/not-all-github-files-cloning-to-my-local-machine-missing-all-example-files/220490 <--- The env file is local and needs to be created locally? 
3. https://www.freecodecamp.org/news/nodejs-custom-env-files-in-your-apps-fa7b3e67abe1/#:~:text=new%20to%20the%20.-,env%20file.,parses%20them%20to%20the%20environment. <--- Apparently used in conjunction with node ? 

- What have you tried? Tell us everything you have tried. npm start, project runs successfully without. 

- Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰. N/A

**Comment by ajawashington** [comment, marked as answer]
The web-pack template for Bear Watcher doesn't have the same configuration, so you don't need the .env.
EDITED: Great Question!

**Reply by caseywalker** [reply]
Thank you! Just wanted to confirm before I got too far into the project.
